index.html

```diff
@@ -1,20 +1,290 @@
+						Have the time to share your thoughts and opinions
+						with experts for each topic.
+					</p>
+				</div>
+			</div>
+
+			<!-- program card 4  -->
+			<div class="card-item">
+				<img src="./public/images/icon4.png" alt="Workshop Icon" />
+				<div class="activity">
+					<h3>Workshop</h3>
+				</div>
+				<div>
+					<p>
+						Try creating your own work using open source license
+						rather than just watching at it.
+					</p>
+				</div>
+			</div>
+
+			<!-- program card 5  -->
+			<div class="card-item">
+				<img src="./public/images/icon5.png" alt="Job Search Icon" />
+				<div class="activity">
+					<h3>CC Ignite</h3>
+				</div>
+				<div>
+					<p>
+						get opportunities to network with CC affiliates
+						around the world, also after the summit.
+					</p>
+				</div>
+			</div>
+		</div>
+		<button type="button">Join Asia Web Conference 2022</button>
+		<a href="./index.html" class="see-program text-center">SEE THE WHOLE PROGRAM</a>
+	</section>
+
+	<!-- speakers section  -->
+	<section id="speakers" class="relative flex flex-col items-center gap-8 py-8 px-4 text-heavy-metal">
+		<h2
+			class="speakers-title relative text-center text-2xl font-medium after:absolute after:left-[calc(50%-48px/2)] after:-bottom-3 after:h-[2px] after:w-12 after:bg-coral-red">
+			Featured Speakers
+		</h2>
+		<div class="grid w-full max-w-480px grid-cols-1 gap-4 px-6 md:my-20 max-w-full grid-cols-2 gap-11"
+			id="speakers-card-container"></div>
+		<button id="speakers-btn" class="md:hidden w-full border-2 px-4 py-3 text-metal-grey">
+			<span>MORE</span>
+			<i class="fa-solid fa-chevron-down ml-2 rotate-180 text-coral-red"></i>
+		</button>
+	</section>
+
+		<!-- partner section  -->
+	<section id="partner" class="relative flex flex-col items-center bg-heavy-metal py-8 px-4 mobile-hidden">
+      <h2 class="partner-title relative text-center text-2xl font-medium text-metal-grey after:absolute after:left-[calc(50%-48px/2)] after:-bottom-3 after:h-[2px] after:w-12 after:bg-coral-red">
+        Partner
+				<img class="indicator" src="./public/images/indicator.svg" alt="Line Break" />
+      </h2>
+      <div class="partners-items-container mt-5 flex w-75 flex-wrap items-center justify-center gap-4">
+        <div class="partners-img-container max-w-120px">
+          <img src="./public/images/partner-mozilla.png" alt="mozilla">
+        </div>
+        <div class="partners-img-container max-w-120px">
+          <img src="./public/images/partner-google.png" alt="google">
+        </div>
+        <div class="partners-img-container max-w-120px">
+          <img src="./public/images/partner-naver.png" alt="naver">
+        </div>
+        <div class="partners-img-container max-w-120px">
+          <img src="./public/images/partner-daumkakao.png" alt="daumkakao">
+        </div>
+        <div class="partners-img-container max-w-120px">
+          <img src="./public/images/partner-airbnb.png" alt="airbnb">
+        </div>
+      </div>
```

Comment:
- [ ] Kindly make sure to list the partners using the `ul` tag and make sure each partner is wrapped into the `li` in order to follow the best HTML practices.